cypress/integration/articleFlows/postModerationTools.spec.js

```diff
@@ -0,0 +1,31 @@
+describe('Moderation Tools for Posts', () => {
+  beforeEach(() => {
+    cy.testSetup();
+  });
+
+  it('should load moderation tools on a post for a trusted user', () => {
+    cy.fixture('users/trustedUser.json').as('user');
+    cy.get('@user').then((user) => {
+      cy.loginAndVisit(user, '/').then(() => {
```

Comment:
I considered hard-coding an URL for a post, but it would be different every time when the E2E tests would run.